fcrepo-auth-common/src/main/java/org/fcrepo/auth/common/FedoraUserSecurityContext.java

```diff
@@ -15,7 +15,7 @@
  */
 package org.fcrepo.auth.common;
 
-import static org.fcrepo.kernel.api.FedoraJcrTypes.JCR_CONTENT;
+import static org.modeshape.jcr.api.JcrConstants.JCR_CONTENT;
```

Comment:
It is confused. I had forgotten about the relationship between authN and MODE. We can alter the title to be more precise and therefore accurate. Did I miss anything other than authN?